lib/CL/pocl_llvm_dynlib.cc

```diff
@@ -64,6 +64,35 @@ void *pocl_dynlib_symbol_address(void *, const char *SymbolName) {
 }
 
 const char *pocl_dynlib_pathname(void *Address) {
+#ifdef _WIN32
+  HMODULE hm;
```

Comment:
C++ sources follow LLVM style where variables should have Pascal casing.
```suggestion
  HMODULE Hm;
```